internal/state/state.go

```diff
@@ -239,18 +240,10 @@ func (state State) MakeBlock(
 	// Build base block with block data.
 	block := types.MakeBlock(height, txs, lastCommit, evidence)
 
-	// Set time.
-	var timestamp time.Time
```

Comment:
So, being a little pick and considering the next step (supporting also BFT Time).

Reply:
Should we set `timestamp` to `cmttime.Now()` here, while keeping L253 unchanged?